modules/lsm/lsm_module_iface.c

```diff
@@ -166,6 +177,41 @@ _drive_update (UDisksObject   *object,
     }
 }
 
+static gboolean
+_lsm_local_check (UDisksObject *object)
+{
+  /* LsmLocalDisk interface will be available on all disk drive */
+  return TRUE;
```

Comment:
Nice :) I wasn't sure if storaged actually respects the the interface specification. Glad to know it does. Thanks!